atom/browser/api/atom_api_web_contents.cc

```diff
@@ -893,21 +939,13 @@ void WebContents::DidFinishNavigation(
 void WebContents::SecurityStyleChanged(
     content::SecurityStyle security_style,
     const content::SecurityStyleExplanations& explanations) {
-    std::string type = "unknown";
-    switch (security_style) {
-      case content::SECURITY_STYLE_UNAUTHENTICATED:
-      case content::SECURITY_STYLE_AUTHENTICATION_BROKEN:
-        type = "insecure";
-        break;
-      case content::SECURITY_STYLE_WARNING:
-        type = "warning";
-        break;
-      case content::SECURITY_STYLE_AUTHENTICATED:
-        type = "secure";
-        break;
-      default: break;
+    if (explanations.ran_insecure_content) {
+      Emit("security-style-changed", "active-mixed-content");
```

Comment:
@bridiver, we have `did-display-insecure-content` for passive mixed content and `did-run-insecure-content` for active one. https://github.com/brave/electron/blob/master/atom/renderer/content_settings_client.cc#L408-L424